scripts/audit-spark-3.2.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# This script generates the commits that went in Apache Spark for audit.
+# Audit is required to evaluate if the code needs to be updated based
+# on new commits merged in Apache Spark. This currently audits changes for
+# Spark-3.2 (master branch).
+# Arguments:
+#   lastcommit - File which contains the latest commit hash when this script ran last.
+#   basebranch - branch in Apache Spark for which commits needs to be audited.
+#                Currently it's master as Spark-3.2 branch is not cut yet.
+
+
+set -ex
+
+lastcommit=""
+basebranch="master"
+REF=${REF:-"main"}
+REF=main
+while getopts v:a: flag
+do
+  case "${flag}" in
+      v) lastcommit=${OPTARG};;
+      b) basebranch=${OPTARG};;
+  esac
+done
+
+SPARK_TREE="$HOME/spark-3.2/spark"
```

Comment:
can this just be pulled from `$SPARK_HOME`??

Reply:
Agreed. Updated it to $SPARK_HOME